launchpad-guaranteed-tickets-v2/src/guaranteed_tickets_init.rs

```diff
@@ -34,13 +34,16 @@ pub trait GuaranteedTicketsInitModule:
         address_number_pairs: MultiValueEncoded<
             MultiValue3<ManagedAddress, usize, ManagedVec<GuaranteedTicketInfo>>,
         >,
-    ) {
+    ) -> (usize, usize) {
```

Comment:
Don't return tuples, especially if they're both of the same type. Nobody ever knows what these mean without looking at this function at the return statement. It's also very easy to mix up the order and end up with weird bugs.

Return a struct with named fields.

Reply:
You're right. I've updated the code.